src/main/java/carpet/script/EntityEventsGroup.java

```diff
@@ -108,6 +109,25 @@ public void addEvent(Event type, ScriptHost host, FunctionValue fun, List<Value>
                 );
             }
         };
+        public static final Event ON_MOVE = new Event("on_move", 3)
+        {
+            @Override
+            public List<Value> makeArgs(Entity entity, Object... providedArgs)
+            {
+                return Arrays.asList(
+                        new EntityValue(entity),
+                        ListValue.fromTriple((Double) providedArgs[0],
+                        (Double) providedArgs[1],
+                        (Double) providedArgs[2]),
+                        ListValue.fromTriple((Double) providedArgs[3],
+                        (Double) providedArgs[4],
+                        (Double) providedArgs[5]),
+                        ListValue.fromTriple((Double) providedArgs[6],
+                        (Double) providedArgs[7],
+                        (Double) providedArgs[8])
```

Comment:
Maybe (and this is just my opinion) you could make it more readable by indenting the arguments of the `ListValue.fromTriple` so we can see which arguments are in that and which are in the `Arrays.asList`

Reply:
I like the idea. I'm kinda new to github so not sure if I should commit that or attach a suggestion here, so I'll do the latter and see if it works lol. If it doesn't just reply to this.
```suggestion
                        ListValue.fromTriple((Double) providedArgs[0],
                            (Double) providedArgs[1],
                            (Double) providedArgs[2]),
                        ListValue.fromTriple((Double) providedArgs[3],
                            (Double) providedArgs[4],
                            (Double) providedArgs[5]),
                        ListValue.fromTriple((Double) providedArgs[6],
                            (Double) providedArgs[7],
                            (Double) providedArgs[8])
```